src/main/java/io/jenkins/plugins/pipeline_elasticsearch_logs/Utils.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.plugins.pipeline_elasticsearch_logs;
+
+import static java.lang.String.format;
+
+import java.io.IOException;
+import java.util.UUID;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Utils {
+
+    public static void logThrowableHideDetails(Logger logger, Level level, String message, Throwable error) throws IOException {
```

Comment:
The important thing this method does is to raise another exception. That is not made clear from the name.

maybe `logExceptionAndReraiseWithTruncatedDetails` ?